### ISSUE TITLE
Working with sockets from wsgi app

### ISSUE THREAD

**Issue by Qwasser** [miguelgrinberg/python-socketio#24]
Hello!
I am trying to work with sockets from django app. I don't understand how can I possibly access to sockets from django. For example, I have request in django that creates some obkects and I want tu brodcast this object to all of the clients. Could you give a hint, please?

**Comment by miguelgrinberg** [comment]
Unfortunately there is no integration with Django at this point, so you will need to write the glue code that connects your Django app with the SocketIO server. The example in this repository shows how you can integrate SocketIO with a Flask server. This should translate fairly easily into Django. If you do make a Django based example app I would love to include it here as an example, by the way.

Another alternative would be to create the SocketIO server separately from your Django app, and then have the Django app connect to it via a message queue to pass on updates.

**Comment by sillygod** [comment]
@Qwasser 

Hello ~

I've written a example with django and make a pull request. Maybe you can see the [pull request here](https://github.com/miguelgrinberg/python-socketio/pull/30)

Regards,
Jing

**Comment by eddyekofo94** [comment]
Integration with a Pyramid app, any examples yet?
Thank you.

**Comment by miguelgrinberg** [comment]
Not yet. I would happily accept one as a contribution if you end up doing it yourself.

**Comment by quietcoolwu** [comment]
@miguelgrinberg Do you mean the example used by js code in `index.html`?

**Comment by miguelgrinberg** [comment]
@quietcoolwu see the example for Django: https://github.com/miguelgrinberg/python-socketio/tree/master/examples/server/wsgi/django_example

**Comment by quietcoolwu** [comment]
@miguelgrinberg thanks for your quick response!
yeah, the `javascript` code in `index.html` in this dir is a must-need now? If we removed the html, will python-socketio embedded in `django` still work?
I found that you claimed `Unfortunately there is no integration with Django at this point, so you will need to write the glue code that connects your Django app with the SocketIO server`, I wonder do we still need to hack the django now?
